src/custom/hooks/web3.ts

```diff
@@ -88,7 +88,7 @@ export function useEagerConnect() {
     if (!active) {
       const latestProvider = localStorage.getItem(STORAGE_KEY_LAST_PROVIDER)
 
-      // if there is no last saved provider set tried state to true
+      // If there is no last saved provider set tried state to true
```

Comment:
wut

Reply:
Not my changes.

For context: When reviewing Nenad's original PR, there were comments about the case of the comments.
Keep in mind these were all coming from Uniswap.
Anyway, @nenadV91 was very kind and did abide by the requests.